### PR TITLE
Fix issue #133 for correct display of IPv4 and IPv6

### PR DESCRIPTION
**Pull request by vkatsuba**
Issue https://github.com/travelping/ergw/issues/133
From the example of task, it can be seen that sometimes, for some reason, the addresses can have binary format like `<<10,65533,126,65533>>` etc. which can break the display of the address and will displayed something like `<<"\ný~ý">>` etc.

### CODE REVIEW

File: src/ergw_sx_node.erl [hunk 1]
```diff
@@ -450,7 +450,7 @@ handle_event({call, From},
 	     _State,
 	     #data{pfcp_ctx = #pfcp_ctx{seid = #seid{dp = SEID}} = PCtx, tdf = Tdf}) ->
     {ok, {tdf, VRF}} = ergw_pfcp:find_urr_by_id(Id, PCtx),
-    ?LOG(debug, "Sx Node TDF Report on ~p for UE IPv4 ~p IPv6 ~p", [VRF, IP4, IP6]),
+    ?LOG(debug, "Sx Node TDF Report on ~p for UE IPv4 ~p IPv6 ~p", [VRF, ergw_inet:bin2ip(IP4), ergw_inet:bin2ip(IP6)]),
```
Comment (RoadRunnr) [inline src/ergw_sx_node.erl:453]: If you insist on formating IPs for debug logging, then format to something human readable and not from binary to Erlang tuples.

**Review by RoadRunnr — CHANGES_REQUESTED** [review]

File: src/ergw_sx_node.erl (hunk 1 above)
Comment (vkatsuba) [reply src/ergw_sx_node.erl:453]: I was think that `{x, x, x, x}` will be normal. What is your preferred format?

Comment (RoadRunnr) [reply src/ergw_sx_node.erl:453]: `inet:ntoa/1`

Comment (vkatsuba) [reply src/ergw_sx_node.erl]: Done

**Review by fholzhauser — APPROVED** [review]

**Review by RoadRunnr — APPROVED** [review]

**Pull request merged**